infra/base-images/base-runner/coverage

```diff
@@ -16,12 +16,18 @@
 ################################################################################
 cd $OUT
 
+# copy version/language-specific coverage tools if any
+cp $OUT/llvm-cov /usr/local/bin || true
```

Comment:
Not a fan of this. We copy llvm-cov and llvm-profdata into the build for rust and swift and then copy those from build into /usr/local/bin if they exist?
Here's a way I like more (assuming we end up using multiple versions of clang): Copy llvm-cov and llvm-profdata into build for all fuzzers no matter the language, then instead of copying those into /usr/local/bin, use them directly.